community/git.md

```diff
@@ -0,0 +1,19 @@
+# Sharing mods with git
+
+Author(s): code3z (@code3 on the forum)
+
+Git is a tool often used by developers to share and collaborate on great programs and docs (including this guide!). It allows developers to make individual changes to the main set of files and merge changes together.
+
+By using it to share your mods, you can make it easier to download a copy of all your mods, upload multiple files, put a license on all your mods, and allow others to suggest changes to make your mods better. If the Git provider has features like Issues and Projects, it can make it easier to collect bugs and plan new mods. You'll need an account on a git server provider such as Github (closed-source, free, most popular) or Gitlab (open-source, free, also popular). This guide won't teach you all about using git, but you can learn that using these guides:
+ -
+ -
+
+## Community Modding Git repositories
+
+Here are some Modding Git Repositories that members of the community created to upload their mods, you may find some useful things there:
+  - https://github.com/LonMcGregor/VivaldiMods
+  - https://github.com/luetage/vivaldi_modding
+  - https://github.com/sjudenim/Mods-for-Vivaldi
+  - https://github.com/GabeVilela/Vivaldi-GX
+
+If you have a modding repository of your own, you can add it with a pull request (even your repo is on Gitlab or another provider).
```

Comment:
‘send a pull request or let us know’, & I’d take out the note about GitLab or rephrase it not to sound like they are less welcome.